File: RailgunVB/Commands/Utilities/Ping.vb
Imports Discord
Imports Discord.Commands
Imports Discord.WebSocket
Imports RailgunVB.Core

Namespace Commands.Utilities
    
    <Group("ping")>
    Public Class Ping
        Inherits SystemBase
        
        Private ReadOnly _client As DiscordShardedClient
        
        Public Sub New(client As DiscordShardedClient)
            _client = client
        End Sub
    
        <Command>
        Public Function PingAsync() As Task
            Return ReplyAsync($"Discord Client Latency : {Format.Bold(_client.Latency.ToString())}ms")
        End Function
        
    End Class
    
End NameSpace

File: Railgun/Commands/Ping.cs
using System.Threading.Tasks;
using Discord;
using Discord.WebSocket;
using Finite.Commands;
using Railgun.Core;

namespace Railgun.Commands
{
    [Alias("ping")]
    public class Ping : SystemBase
    {
        private readonly DiscordShardedClient _client;

        public Ping(DiscordShardedClient client) => _client = client;

        [Command]
        public Task PingAsync()
            => ReplyAsync($"Discord Client Latency : {Format.Bold(_client.Latency.ToString())}ms");
    }
}